src/sentry/tsdb/redis.py

```diff
@@ -96,13 +96,15 @@ def __init__(self, prefix='ts:', vnodes=64, **options):
         self.cluster, options = get_cluster_from_options(self, options)
         self.prefix = prefix
         self.vnodes = vnodes
+        self.enable_frequency_sketches = options.pop('enable_frequency_sketches', False)
         super(RedisTSDB, self).__init__(**options)
 
     def validate(self):
         logger.debug('Validating Redis version...')
+        version = Version((2, 8, 18 if self.enable_frequency_sketches else 9))
```

Comment:
imo I'd rather do `Version((2, 8, 18)) if self.enabled_frequency_sketches else Version((2, 8, 9))`

It reads more clearly, and I can see someone messing up if they bumped `(2, 9, 9)` or something, since it'd affect both.